protocol/protocol.tex

```diff
@@ -15087,6 +15087,11 @@
           \MUST be considered invalid when imported.
           Notes have been added in \crossref{saplingpaymentaddrencoding} and
           \crossref{saplinginviewingkeyencoding} calling out these changes.
+    \item In \crossref{decryptovk}, retrospectively enforce the canonicity check on
+          $\DiversifiedTransmitPublic$. This has no effect on consensus because only small-order
+          \jubjubCurve points have \nonCanonicalPoint encodings, and so the check that
+          returns $\bot$ if $\DiversifiedTransmitPublic \not\in \SubgroupJstar$ would catch
+          all such cases.
```

Comment:
```suggestion
          $\DiversifiedTransmitPublic$. This has no effect on consensus relative to the
          previous version, because only small-order
          \jubjubCurve points have \nonCanonicalPoint encodings, and so the check that
          returns $\bot$ if $\DiversifiedTransmitPublic \not\in \SubgroupJstar$ would catch
          all such cases.
```

I'm still not fully convinced by this reasoning, but I believe it to in any case be a valid change because any possible consensus divergence was not exercised prior to NU5.